cmd/cat-main.go

```diff
@@ -63,6 +72,9 @@ EXAMPLES:
    3. Concatenate multiple files to one.
       $ {{.HelpName}} part.* > complete.img
 
+   4. Stream a server encrypted object from Amazon S3 cloud storage to standard output.
+      $ {{.HelpName}} --encrypt 'customerspecifiedencryptky32bits' s3/ferenginar/klingon_opera_aktuh_maylotah.ogg
```

Comment:
`...32bits` is a bit misleading, can this end with `...32byte`

Reply:
good idea. This is back to wip for a different approach of taking encryption keys at command line.